Fix: Add PHP 7.1 and 7.2 to build matrix

localheinz opened the pull request:
This PR

* [x] adds PHP 7.1 and 7.2 to the Travis build matrix

localheinz committed:
Fix: Add PHP 7.1 and 7.2 to build matrix

weierophinney committed:
Merge pull request #39 from localheinz/fix/matrix

Fix: Add PHP 7.1 and 7.2 to build matrix

weierophinney commented:
@localheinz I'll take care of the lingering issues in Travis; they're due to some dependencies that require PHP 7.1 (doctrine/instantiator for one), and I know ways to get around this.

weierophinney committed:
Simplifies `.travis.yml` to ensure passage on all environments

- Removes `--ignore-platform-reqs` flag; we need to honor them.
- Removes documentation build requirements; now handled by webhook.

weierophinney commented:
Thanks, @localheinz!

localheinz commented:
Thank you, @weierophinney!
